Source/WebCore/rendering/RenderLayer.cpp

```diff
@@ -1332,6 +1332,15 @@ void RenderLayer::updateTransform()
     if (hasTransform) {
         m_transform->makeIdentity();
         updateTransformFromStyle(*m_transform, renderer().style(), RenderStyle::allTransformOperations());
+        if (!hadTransform && !m_transform->isIdentity()) {
```

Comment:
Hmm, this is unncessary for non-SVG renderers at least.
Also extra work if CSS transforms are used for SVG renderers. I think the new code updating the stacking context state needs to be performed _only_ when we change the transform attr/prop of a SVG renderer + repaint, instead of a full-fledged layout, no?

Reply:
You are right, the location I picked was not very SVG specific, I just did not like adding a method on RenderLayer, but I guess we can't avoid that. New iteration should be better, PTAL :)